src/ralph/accounts/views.py

```diff
@@ -39,17 +117,28 @@ def get_context_data(self, **kwargs):
             'model__category__name', 'model__manufacturer__name',
             'model__name', ('sn', _('Serial Number')), 'invoice_date', 'status'
         ]
+
         if settings.MY_EQUIPMENT_SHOW_BUYOUT_DATE:
             asset_fields += ['buyout_date']
 
         if settings.MY_EQUIPMENT_REPORT_FAILURE_URL:
             asset_fields += ['report_failure']
+
+        if BackOfficeAsset.objects.filter(
```

Comment:
1. use ``.exists()`` instead of ``.count() > 0``
2. move conditions to variables ex. ``warehouse_stocktaking_enabled``, ``region_stocktaking_enabled``